server/bundles/io.cloudbeaver.model/src/io/cloudbeaver/model/rm/local/LocalResourceController.java

```diff
@@ -121,6 +199,9 @@ protected RMLocalProject getWebProject(String projectId, boolean refresh) throws
             RMLocalProject project = projectRegistries.get(projectId);
             if (project == null || refresh) {
                 project = createWebProjectImpl(projectId, new SessionContextImpl(null));
+                if (RMProjectType.SHARED.equals(project.getProjectType())) {
```

Comment:
`equals` on enums is redundant
```suggestion
                if (project.getProjectType() == RMProjectType.SHARED) {
```